dotcom-rendering/src/web/components/Callout/CalloutComponents.tsx

```diff
@@ -103,31 +103,37 @@ const shareCalloutTextStyles = css`
 
 const shareCalloutLinkStyles = css`
 	color: ${brand[500]};
-	border-bottom: 1px solid ${brand[500]};
-	text-decoration: none;
 	font-weight: normal;
-	margin: 0 ${space[1]}px;
+	${textSans.xsmall()}
 `;
 
-const tooltipStyles = css`
+const sharePopupStyles = css`
 	${textSans.xsmall()};
 	position: absolute;
 	display: flex;
 	align-items: center;
 	min-width: 180px;
-	bottom: -20px;
+	transform: translate(100%, 0);
 	background-color: ${neutral[100]};
 	color: ${neutral[7]};
 	font-weight: normal;
-	border-radius: 4px;
+	border-radius: ${space[1]}px;
 	z-index: 1;
 	padding: 0 ${space[1]}px 0 0;
-	box-shadow: 0px 2px 8px rgba(0, 0, 0, 0.5);
+	box-shadow: 0px 0.125rem ${space[2]}px rgba(0, 0, 0, 0.5);
```

Comment:
Do we need to use rem here? since the rest of the values in this box-shadow are all px